RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -399,3 +398,92 @@ void auto_useBlackMonolith()
 	// use monolith
 	visit_url("campground.php?action=monolith");
 }
+
+boolean auto_haveAugustScepter()
+{
+	static item scepter = wrap_item($item[august scepter]);
+	if(auto_is_valid(scepter) && (item_amount(scepter) > 0 || have_equipped(scepter)))
+	{
+		return true;
+	}
+	return false;
+}
+
+void auto_scepterSkills()
+{
+	if(!auto_haveAugustScepter())
+	{
+		return;
+	}
+	//Day 1 skills
+	if(my_daycount() == 1)
+	{
+		//Get some stats
+		if(my_primestat() == $stat[Muscle])
+		{
+			if(canUse($skill[Aug. 12th: Elephant Day!]) && !get_property("_aug12Cast").to_boolean())
+			{
+				use_skill($skill[Aug. 12th: Elephant Day!]);
+			}
+		}
+		if(my_primestat() == $stat[Mysticality])
+		{
+			if(canUse($skill[Aug. 11th: Presidential Joke Day!]) && !get_property("_aug11Cast").to_boolean())
+			{
+				use_skill($skill[Aug. 11th: Presidential Joke Day!]);
+			}
+		}
+		if(my_primestat() == $stat[Moxie])
+		{
+			if(canUse($skill[Aug. 23rd: Ride the Wind Day!]) && !get_property("_aug23Cast").to_boolean())
+			{
+				use_skill($skill[Aug. 23rd: Ride the Wind Day!]);
+			}
+		}
+		if(canUse($skill[Aug. 24th: Waffle Day!]) && !get_property("_aug24Cast").to_boolean())
+		{
+			use_skill($skill[Aug. 24th: Waffle Day!]); //get some waffles to hopefully change some bad monsters to better ones
+		}
+		if(canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean())
+		{
+			use_skill($skill[Aug. 30th: Beach Day!]); //Rollover adventures
+		}
+		if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && ((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small()))
+		{
+			if(!is100FamRun())
+			{
+				handleFamiliar("stat"); //get any familiar equipped if not in a 100% run
```

Comment:
don't believe `handleFamiliar` changes familiar immediately. Suggest changing to `use_familiar($familiar[Mosquito]);` since everyone will have one

Reply:
Hah you say that but then there's the guy who has never put one in their terrarium despite playing for over 10 years and having hundreds of ascensions...

No joke this actually happened. Another person had no Ghuol Whelp (may have been the same person) and I think I recall someone without a Smiling Rat or Volleyball because they had the Barrel Mimic and hadn't been in a Standard restricted path since it left Standard.

Regardless either is fine. People like that deserve their aborts.

Reply:
Yeah, I didn't look into handleFamiliar and it doesn't switch immediately. will do what Alium suggested with the mosquito because as it turns out, I wasn't really paying much attention at the beginning of D1 and now I have a bunch of big rocks